cli.go

```diff
@@ -208,5 +211,23 @@ func (app *TGFApplication) Run() int {
 		Printf("tgf v%s\n", version)
 		return 0
 	}
+
+	if !app.NoUpdate {
+		didUpdate := RunUpdate()
+
+		if didUpdate {
+			Println("tgf updated !")
```

Comment:
Use logging functions, not print

Reply:
I did it because above we are already using `Println` when we running `tgf --current-version`.
Is there a place for both?

Reply:
It's different though. These are logs and when you run `--current-version`, you expect to have an output. You don't want to have logs in your stdout

Reply:
Oh okok

Reply:
From what I've seen, we use mainly `Println`...
So I'm not sure, is `log.Println` okay?

Reply:
You can use `app.Debug`